.infra/Pulumi.prod.yaml

```diff
@@ -28,7 +28,7 @@ config:
       secure: AAABALdzzBMEk2sKYDjFrWsbsdYJDOvfDfmnOmaj6lvRkds8E6SYw1RNgQ/hpCky9W8BeJXFzu2b0W6jD30KkA==
     analyticsUrl: http://analytics-api
     apiConfigFeatureKey: api-config
-    cdcWorkerMaxMessages: 1
+    cdcWorkerMaxMessages: 100
```

Comment:
Will this interfere with order of CDC since now more messages will be run in parallel?